depthai_examples/src/stereo_inertial_publisher.cpp

```diff
@@ -0,0 +1,277 @@
+                                                                                            std::placeholders::_2) , 
+                                                                                            30,
+                                                                                            leftCameraInfo,
+                                                                                            "left");
+            dai::rosBridge::BridgePublisher<sensor_msgs::Image, dai::ImgFrame> rightPublish(rightQueue,
+                                                                                            pnh, 
+                                                                                            rightPubName,
+                                                                                            std::bind(&dai::rosBridge::ImageConverter::toRosMsg, 
+                                                                                            &rightconverter, 
+                                                                                            std::placeholders::_1, 
+                                                                                            std::placeholders::_2) , 
+                                                                                            30,
+                                                                                            rightCameraInfo,
+                                                                                            "right");  
+            rightPublish.addPubisherCallback();
+            leftPublish.addPubisherCallback();
+            ros::spin();
+        }
+    }
+    else{
+        dai::rosBridge::DisparityConverter dispConverter(deviceName + "_right_camera_optical_frame", 880, 7.5, 20, 2000);
+        dai::rosBridge::BridgePublisher<stereo_msgs::DisparityImage, dai::ImgFrame> dispPublish(stereoQueue,
+                                                                                     pnh, 
+                                                                                     std::string("stereo/disparity"),
+                                                                                     std::bind(&dai::rosBridge::DisparityConverter::toRosMsg, 
+                                                                                     &dispConverter, 
+                                                                                     std::placeholders::_1, 
+                                                                                     std::placeholders::_2) , 
+                                                                                     30,
+                                                                                     rightCameraInfo,
+                                                                                     "stereo");
```

Comment:
We can publish disparity aligned with rgb also. Doesn't need to be only with depth.